metadata.yaml

```diff
@@ -46,6 +46,10 @@ requires:
   database:
     interface: postgresql_client
     limit: 1 # Most charms only handle a single PostgreSQL Application.
+  pro-airgapped-server:
+    interface: livepatch-pro-airgapped-server
```

Comment:
name of the relation interface seems wrong. This name implies that this relation is intended for services that require a running livepatch server. What we really mean is that this relation is intended for services that need a running ua-contracts. Perhaps an interface name like "pro-contracts" or just "contracts" (but this may conflict with the contracts service later on)

Reply:
As we discussed before, the relation is just meant to be used by Livepatch, so the name is specific.